### ISSUE TITLE
Old event pages are visible

### ISSUE THREAD

**Issue by JobDoesburg** [svthalia/concrexit#1638]
### Describe the bug
It is possible to visit a page for an old event that has long past and view the event registrations for that event. In my opinion, there is no need for that information to be visible (as it isn't displayed in the calendar anymore either). 

### How to reproduce
1. Visit an old event, like https://thalia.nu/events/3/
2. I can see who has registered for that event

### Expected behaviour
Either give some kind of error, or better, do display the event, but don't display the event registrations (as if the user wasn't logged in)

**Comment by DeD1rk** [comment]
The events should definitely still be visible, and it would be nice if at least the number of registrations is also visible. In the old thaliapp the ability to see old events was added as a feature relatively recently (just around before the early corona days).

**Comment by JobDoesburg** [comment]
> it would be nice if at least the number of registrations is also visible

I think that is the case already when you're not logged in. So yes

**Comment by se-bastiaan** [comment]
I don't entirely understand the reasoning, why should this be hidden exactly? Why is this a bug?

**Comment by JobDoesburg** [comment]
I think it is a bit data leak. There is no reason why the information who was present at an event should be visible to other members years after the event took place.

**Comment by pingiun** [comment]
I agree with Job that if personal data is not neccessary, we shouldn't show it. I think the best solution is to hide the information after a certain time (four weeks?). I've added the api label as well because we shouldn't forget to remove the information from there.
